build.sbt

```diff
@@ -21,7 +21,7 @@ lazy val common = commonRootSettings ++ Seq(
         Seq("-Xfatal-warnings", "-deprecation:false")
       case Some((2, scalaMinor)) if scalaMinor < 11 =>
         // For scala versions 2.10.x
-        Seq("-Xfatal-warnings")
+        Seq()
```

Comment:
Too late. I broke backwards compatability. You'll just have to learn the new way :smile: Don't worry, it's better.